### PR TITLE
Added System,Upvote,Downvote to Note

### PR DESCRIPTION
**Pull request by violet-day**

### CODE REVIEW

**Comment by violet-day** [comment]
ping @svanharmelen, please review

File: notes.go [hunk 1]
```diff
@@ -49,6 +49,9 @@ type Note struct {
 		State     string     `json:"state"`
 		CreatedAt *time.Time `json:"created_at"`
 	} `json:"author"`
+	System bool `json:"system"`
+	Upvote bool `json:"upvote"`
+	Downvote bool `json:"downvote"`
```
Comment (svanharmelen) [inline notes.go]: Please run `gofmt` this file to make sure your code is formatted correctly.

Comment (violet-day) [inline notes.go]: `gofmt` done

**Comment by svanharmelen** [comment]
LGTM! Thanks @violet-day!

**Pull request merged**